nc-news-bh-final-vite/src/components/DeleteComment.jsx

```diff
@@ -0,0 +1,26 @@
+import {useEffect, useState} from 'react'
+import { deleteComment } from '../utils/api.js'
+
+
+const DeleteComment = ({comment_id, setIsDeleting})=>{
+
+        function handleClick(){
+            console.log("yo")
+            console.log(comment_id)
+            setIsDeleting(true)
+            deleteComment(comment_id).then((data)=>{
+                setIsDeleting(false)
```

Comment:
You are setting 'isDeleting' here, but where do you actually use it to display a message to the user? It's not used for that purpose in the 'Comments' component either.

Reply:
this is used as a dependency in the useEffect in the comments component